src/main/kotlin/uk/co/craigbass/pratura/usecase/CurrencySetter.kt

```diff
@@ -0,0 +1,7 @@
+package uk.co.craigbass.pratura.usecase
+
+import uk.co.craigbass.pratura.domain.Currency
```

Comment:
I think they are very different. One is related to admin, the other is related to customer baskets